### DISCUSSION TITLE
Login Failure on Production: 500 Internal Server Error

### DISCUSSION THREAD

**Discussion by SamGynius** [supabase/supabase#30971] [Questions] [answered]
Hi everyone,

I'm encountering a critical issue with my application's login functionality in production. The app is built using Next.js 14 and Supabase, deployed on Netlify with Godaddy DNS. While everything works fine locally, I'm facing consistent login failures when accessing the site through my custom domain (mysite.com).

Key details:
- Environment: Production (Netlify)
- Custom domain: mysite.com
- Stack: Next.js 14, Supabase (latest js client)
- Error: 500 Internal Server Error on login attempt

Symptoms:
1. Users receive a popup error: "An unexpected error occurred. Please try again."
2. Browser console shows multiple "Failed to load resource: the server responded with a status of 500 ()" errors.
3. POST request to https://mysite/user/login results in a 500 Internal Server Error.

What I've tried:
- Verified CORS configuration
- Checked Supabase integration and API keys
- Compared local and production environments
- Reviewed server-side logs (no clear error messages found)

Netlify configuration shows: "Application error: a client-side exception has occurred (see the browser console for more info)", but the login page loads correctly.

I'm at a loss for what could be causing this discrepancy between local and production environments. Any insights, troubleshooting steps, or potential solutions would be greatly appreciated.

Thank you in advance for your help!

**Comment by GaryAustin1** [comment, marked as answer]
Normally a 500 error on login from Supabase is a user trigger function on auth.users or SMTP error occurring.   But there should be an error in the Dashboard Auth and/or Postgres logs.

**Comment by SamGynius** [comment]
Thanks @GaryAustin1! I resolved it.
